mobile/src/main/java/com/boswelja/devicemanager/managespace/ManageSpaceActivity.kt

```diff
@@ -204,7 +201,7 @@ class ManageSpaceActivity : BaseToolbarActivity() {
     private fun doSettingsReset() {
         setButtonsEnabled(false)
         coroutineScope.launch(Dispatchers.IO) {
-            val registeredWatches = watchManager.getRegisteredWatches()
+            val registeredWatches = watchManager.registeredWatches.value!!
```

Comment:
There's a potential bug here, if registeredWatches hasn't been observed then it likely won't have been updated, making the list null or empty.